frontend/src/routes.js

```diff
@@ -40,10 +40,25 @@ app.get('/job-roles/:id', async (req, res) => {
     const bandResponse = await fetch(api_url+'bands/'+roleData.bandID,{method:'GET',headers:{}})
     const bandData = await bandResponse.json();
     console.log({role: roleData, band: bandData})
-    res.render('jobRolePage', {role: roleData, band: bandData}); 
+    res.render('jobRolePage', {role: roleData, band: bandData});
 
 });
 
-app.listen(6555, function() { 
+app.get('/capabilities', async (req, res) => {
+    const response = await fetch('http://localhost:8080/api/capabilities',{method:'GET',headers:{}})
+    const caps = await response.json();
+    console.log(caps);
+    res.render('capabilities', {items: caps});
+
+});
+
+app.get('/capabilities/:id', async (req, res) => {
+    var id = req.params.id;
+    const response = await fetch(api_url+'capabilities/'+id,{method:'GET',headers:{}})
```

Comment:
The URL here may be more readable with string interpolation
```suggestion
    const response = await fetch(`${api_url}capabilities/${id}`,{method:'GET',headers:{}})
```
(I also recommend you don't include the trailing slash in your `API_URL`, in which case add it to the string above.)